tests/Browser/Actions/Test.php

```diff
@@ -119,4 +119,34 @@ public function test()
             ;
         });
     }
+
+    public function test_button_dom_update_click_event()
+    {
+        $this->browse(function ($browser) {
+            Livewire::visit($browser, Component::class)
+                /**
+                 * sibling buttons
+                 */
+                ->press('@show.button.actions')
+                ->waitForLivewire()->pause(100)->assertPresent('@button.with-actions')
+                ->press('@button.with-click')
+                ->waitForLivewire()->pause(100)->assertSeeIn('@output', 'button with wire:clicked got triggered')
+            ;
+        });
+    }
+
+    public function test_button_dom_update_click_event_with_id()
```

Comment:
Due to **performance** reasons, we structured all browser **tests** to have **only one** test **method** per file and just chain all "real" tests within it.

To separate them we use doc-blocks and if needed a `->refresh()` call.